demos/python_demos/multi_camera_multi_person_tracking/mc_tracker/sct.py

```diff
@@ -0,0 +1,405 @@
+
+    def _continue_tracks(self, detections, features):
+        active_tracks_idx = []
+        for i, track in enumerate(self.tracks):
+            if track['timestamps'][-1] >= self.time - self.continue_time_thresh:
+                active_tracks_idx.append(i)
+
+        occluded_det_idx = []
+        for i, det1 in enumerate(detections):
+            for j, det2 in enumerate(detections):
+                if i != j and self._ios(det1, det2) > self.detection_occlusion_thresh:
+                    occluded_det_idx.append(i)
+                    features[i] = None
+                    break
+
+        cost_matrix = self._compute_detections_assignment_cost(active_tracks_idx, detections, features)
+
+        assignment = [None for _ in range(cost_matrix.shape[0])]
+        if cost_matrix.size > 0:
+            row_ind, col_ind = linear_sum_assignment(cost_matrix)
+            for i, j in zip(row_ind, col_ind):
+                idx = active_tracks_idx[j]
+                if cost_matrix[i, j] < self.match_threshold and \
+                        self._check_velocity_constraint(self.tracks[idx], detections[i]) and \
+                        self._giou(self.tracks[idx]['boxes'][-1], detections[i]) > self.track_detection_iou_thresh:
+                    assignment[i] = j
+
+            for i, j in enumerate(assignment):
+                if j is not None:
+                    idx = active_tracks_idx[j]
+                    self.tracks[idx]['boxes'].append(detections[i])
+                    self.tracks[idx]['timestamps'].append(self.time)
+                    self.tracks[idx]['features'].append(features[i])
+                    if features[i] is not None:
+                        self.tracks[idx]['f_cluster'].update(features[i])
+                        if self.tracks[idx]['avg_feature'] is None:
+                            self.tracks[idx]['avg_feature'] = np.zeros(features[i].shape)
+                        self.tracks[idx]['avg_feature'] += (features[i] - self.tracks[idx]['avg_feature']) / \
+                                                            len(self.tracks[idx]['features'])
+                    else:
+                        self.tracks[idx]['avg_feature'] = None
+
+        return assignment
+
+    def _merge_tracks(self):
+        clear_tracks = []
+        for track in self.tracks:
+            # remove too old tracks
+            if track['timestamps'][-1] < self.time - self.track_clear_thresh:
+                track['features'] = []
+                self.history_tracks.append(track)
+                continue
+            # remove too short and outdated tracks
+            if track['timestamps'][-1] < self.time - self.continue_time_thresh \
+                    and len(track['timestamps']) < self.time_window:
+                self.global_id_releaser(track['id'])
+                continue
+            clear_tracks.append(track)
+        self.tracks = clear_tracks
+
+        distance_matrix = 10*np.eye(len(self.tracks), dtype=np.float32)
```

Comment:
I suggest giving this `10` a name like `THE_BIGGEST_DISTANCE` or `DONT_ASSIGN`.

Reply:
Fixed

Reply:
There is another 10 below: `distance_matrix[i, j] = 10`. I think it should also be replaced.

Reply:
Fixed this one too